phpcs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<ruleset name="KameleonProject">
+    <arg name="basepath" value="."/>
+    <arg name="extensions" value="php"/>
+    <arg name="parallel" value="45"/>
+    <arg name="cache" value=".phpcs-cache"/>
+    <arg name="colors"/>
+
+    <!-- Ignore warnings and show progress of the run -->
+    <arg value="nps"/>
+
+    <!-- Folders that will be track -->
+    <file>config</file>
+    <file>src</file>
+    <file>tests</file>
+
+    <rule ref="LeroyMerlin"/>
+
+    <!-- Custom rules for this project -->
+    <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration">
+        <properties>
+            <property name="enableEachParameterAndReturnInspection" value="true"/>
+            <property name="usefulAnnotations" type="array">
+                <element value="@dataProvider"/>
+                <element value="@depends"/>
+                <element value="@group"/>
+                <element value="@link"/>
+                <element value="@see"/>
+                <element value="@test"/>
```

Comment:
As the maintenance of this project, @brnbp, like the following syntax for tests

```php
/** @test */
public function it_should_be_tested(): void
```

this annotations must be added to this array.

Reply:
I disagree. We should use the same pattern.

Reply:
If we are really doing this, we should avoid exceptions

Reply:
@tomasvotruba Do you think we can create a https://github.com/rectorphp/rector for this case?

```diff
-/** @test */
-public function it_should_be_tested(): void
+public function testItShouldBeTested(): void
```

Reply:
@carusogabriel Of course :+1: